app/views/items/index.html.erb

```diff
@@ -1,80 +1,76 @@
   <div class="center jumbotron">
```

Comment:
I just rearranged the HTML here, it's all the same as before aside from adding CSS class and ID tags